lib/cambiatus/shop/product.ex

```diff
@@ -52,6 +63,35 @@ defmodule Cambiatus.Shop.Product do
     |> put_assoc(:community, attrs.community)
     |> foreign_key_constraint(:creator_id)
     |> foreign_key_constraint(:community_id)
+    |> validate_community_shop_enabled()
+    |> validate_track_stock_units()
+  end
+
+  def validate_community_shop_enabled(changeset) do
+    changeset
+    |> get_field(:community_id)
+    |> Commune.get_community()
+    |> case do
+      {:ok, community} ->
+        if Map.get(community, :has_shop),
+          do: changeset,
+          else: add_error(changeset, :community_id, "news is not enabled")
```

Comment:
Should this be "shop is not enabled"?
```suggestion
          else: add_error(changeset, :community_id, "shop is not enabled")
```

Reply:
nice catch! thanks bro!